apps/dashboard/src/@3rdweb-sdk/react/hooks/useVote2.ts

```diff
@@ -0,0 +1,260 @@
+import type { Vote } from "@thirdweb-dev/sdk";
```

Comment:
This file will be stashed.
I have decided to make this PR smaller by focusing on the SDK extensions.

Will start migrating the v4 Vote code on dashboard once this PR's released